### ISSUE TITLE
SMTP server

### ISSUE THREAD

**Issue by DubThink** [tobydragon/GameShell#45]
I believe I can set up the logging code to send a message to an smtp server with the log file either as plaintext or an attachment. However, pretty much all the major email providers require authentication, and given python's plaintext nature, I'd rather not include an email account password in the software. Do we have the capability to host an smtp server?

**Comment by tobydragon** [comment]
I don't think this is a viable option, at least for this semester. I think we can create a throw-away gmail account and put thte password in plain text and this is not a huge problem for this small experiment.  If you want to investigate further, I think this link might be helpful:
http://stackoverflow.com/questions/9763455/how-to-send-a-mail-directly-to-smtp-server-without-authentication

**Comment by DubThink** [comment]
Ok, sounds good. Thank you for the link; it looks like it should be helpful.

**Comment by DubThink** [comment]
Email reporting is fully functioning using the dedicated email ICCSresearch1@gmail.com
